sdk/template/template/package.json

```diff
@@ -39,17 +39,17 @@
     "extract-api": "tsc -p . && api-extractor run --local",
     "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"samples-dev/**/*.ts\" \"*.{js,json}\"",
     "generate:client": "autorest --typescript ./swagger/README.md",
-    "integration-test:browser": "karma start --single-run",
-    "integration-test:node": "nyc mocha -r esm --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 5000000 --full-trace \"dist-esm/test/{,!(browser)/**/}/*.spec.js\"",
+    "integration-test:browser": "dev-tool run test:browser",
```

Comment:
`dev-tool run test:browser` is a different script than the `test:browser` in line 48 right? Not a big deal but I was originally confused thinking this would end up in an infinite loop of "test:browser" -> "integration-test:browser" -> "test:browser".

I wonder if we can have a different name in dev-tool?